aws-python/tests/test_infra.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+from typing import Awaitable, TypeVar
+
+import pulumi
+
+
+# Test helper to convert a Pulumi Output to an Awaitable.
+# This should only be used in tests.
+T = TypeVar("T")
+def awaitable(output: pulumi.Output[T]) -> Awaitable[T]:
```

Comment:
Maybe `to_awaitable` ? `await awaitable(...)` reads a bit clumsy, and I find it a bit too close to uppercase `Awaitable`. I don't feel very strongly about this though.